src/main/java/uk/gov/companieshouse/confirmationstatementapi/configuration/ConfirmationStatementServiceEligibilityConfig.java

```diff
@@ -30,20 +32,25 @@
     @Value("${WEB_FILING_COMPANY_TYPES}")
     Set<String> webFilingCompanyTypes;
 
+    @Value("${FEATURE_FLAG_OFFICER_VALIDATION_01062021}")
+    Boolean officerValidationFlag;
```

Comment:
If you assign this a default value. Then you can set the type to the primitive `boolean` instead of the object `Boolean`
`@Value("${FEATURE_FLAG_OFFICER_VALIDATION_01062021:true}")`